src/applications/mhv/medical-records/tests/actions/allergies.unit.spec.js

```diff
@@ -33,19 +32,25 @@ describe('Get allergies action', () => {
 
 describe('Get allergy action', () => {
   it('should dispatch a get details action', () => {
-    const mockData = allergy;
+    const mockData = { id: '1', title: 'Sample Note' };
     mockApiRequest(mockData);
     const dispatch = sinon.spy();
-    return getAllergyDetails('3106')(dispatch).then(() => {
-      expect(dispatch.firstCall.args[0].type).to.equal(Actions.Allergies.GET);
+    return getAllergyDetails('1', [{ id: '1', title: 'Sample Note' }])(
+      dispatch,
+    ).then(() => {
+      expect(dispatch.firstCall.args[0].type).to.equal(
+        Actions.Allergies.GET_FROM_LIST,
+      );
     });
   });
```

Comment:
Rather than modify the existing test, which is still testing valid functionality, I would add a new test to handle your new GET_FROM_LIST case. e.g. (`it('should dispatch a details action and pull from the list', ...)`)

Same comment for all of the modified detail tests..